frontend/src/components/board/Toolbar.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { ChangeEvent, useEffect, useRef } from "react";
+import { Input } from "@/components/ui/input";
+
+import { Bold, Italic, Underline } from "lucide-react";
+import StyledLabel from "@/components/Label";
+import { Toggle } from "@/components/ui/toggle";
+import { Button } from "../ui/button";
+interface ToolBarProps {
+  selectedObjectStyles?: { [key: string]: any } | null;
+  onStyleChange?: (styles: { [key: string]: any }) => void;
+  activeItem: string | null;
+  handleAddImageByUrl: (url: string) => void;
+}
+
+const ToolBar: React.FC<ToolBarProps> = ({
+  selectedObjectStyles,
+  onStyleChange,
+  activeItem,
+  handleAddImageByUrl,
+}) => {
+  const urlRef = useRef<HTMLInputElement>(null);
+
+  const handleChange =
+    (key: string) => (event: ChangeEvent<HTMLInputElement>) => {
+      const value =
+        event.target.type === "number"
+          ? parseInt(event.target.value, 10)
+          : event.target.value;
+      if (onStyleChange) {
+        onStyleChange({ [key]: value });
+      }
+    };
+
+  const renderInput = (
+    styleKey: string,
+    label: string,
+    type: string = "text"
+  ) => {
+    const isColor = type === "color";
+    let inputClasses = "w-[5.5em]";
+    if (isColor) {
+      inputClasses = "w-5 h-5 p-0  rounded-full";
+    }
+    return (
+      <div className={`flex items-center space-x-2 border-r pr-4`}>
+        <StyledLabel htmlFor={styleKey}>{label}</StyledLabel>
+        <Input
+          type={type}
+          id={styleKey}
+          className={inputClasses}
+          value={
+            selectedObjectStyles ? selectedObjectStyles[styleKey] || "" : ""
+          }
+          onChange={handleChange(styleKey)}
+        />
+      </div>
+    );
+  };
+
+  const onBoldClick = () => {
+    if (onStyleChange) {
+      onStyleChange({
+        fontWeight:
+          selectedObjectStyles?.fontWeight === "bold" ? "normal" : "bold",
+      });
+    }
+  };
+
+  const onItalicClick = () => {
+    if (onStyleChange) {
+      onStyleChange({
+        fontStyle:
+          selectedObjectStyles?.fontStyle === "italic" ? "normal" : "italic",
+      });
+    }
+  };
+
+  const onUnderlineClick = () => {
+    if (onStyleChange) {
+      onStyleChange({
+        underline: selectedObjectStyles?.underline === false ? true : false,
```

Comment:
Let's keep it consistent
```suggestion
        underline: 
          selectedObjectStyles?.underline === false ? true : false,
```

Reply:
Actually, it can be written as selectedObjectStyles?.underline === false. Idk how I missed that.